funcbench/main.go

```diff
@@ -205,26 +206,19 @@ func (c *gitClient) checkoutPR(num int) error {
 }
 
 func (c *gitClient) revertPRChanges() error {
-	return filepath.Walk(os.Getenv("GITHUB_WORKSPACE"), func(path string, info os.FileInfo, err error) error {
+	return filepath.Walk(filepath.Join(os.Getenv("GITHUB_WORKSPACE"), c.repo), func(path string, info os.FileInfo, err error) error {
 		if err != nil {
 			return err
 		}
-		if info.IsDir() || strings.Contains(path, ".git/") || strings.HasSuffix(info.Name(), "test.go") {
-			return nil
+		if info.IsDir() && info.Name() == ".git" {
+			return filepath.SkipDir
 		}
```

Comment:
I do not think this `if` statement is necessary. AFAIK, it will skip only `.git` directory, but not files inside the `.git` directory, and you already covered the case of skipping files inside `.git` directory since, AFAIK there are no `.go` file in the `.git` directory.

Reply:
yes you are right I tested it and it still scans the files inside that dir. I though it would skip the entire folder but unfortunately it doesn's.
https://play.golang.org/p/pPAVotsvLs8

Removed it.